nextjsUIapp/src/app/admin/profile/page.tsx

```diff
@@ -14,96 +14,30 @@ export default  async function Profile(){
-                                  <path stroke="currentColor" strokeLinecap="round" strokeLinejoin="round"
-                                        strokeWidth="2" d="M9 1v16M1 9h16"/>
-                              </svg>
-                          </p>
-                      </div>
-                      <div className="flex items-center justify-center rounded bg-gray-50 h-28 dark:bg-gray-800">
-                          <p className="text-2xl text-gray-400 dark:text-gray-500">
-                              <svg className="w-3.5 h-3.5" aria-hidden="true" xmlns="http://www.w3.org/2000/svg"
-                                   fill="none" viewBox="0 0 18 18">
-                                  <path stroke="currentColor" strokeLinecap="round" strokeLinejoin="round"
-                                        strokeWidth="2" d="M9 1v16M1 9h16"/>
-                              </svg>
-                          </p>
-                      </div>
-                      <div className="flex items-center justify-center rounded bg-gray-50 h-28 dark:bg-gray-800">
-                          <p className="text-2xl text-gray-400 dark:text-gray-500">
-                              <svg className="w-3.5 h-3.5" aria-hidden="true" xmlns="http://www.w3.org/2000/svg"
-                                   fill="none" viewBox="0 0 18 18">
-                                  <path stroke="currentColor" strokeLinecap="round" strokeLinejoin="round"
-                                        strokeWidth="2" d="M9 1v16M1 9h16"/>
-                              </svg>
-                          </p>
-                      </div>
-                      <div className="flex items-center justify-center rounded bg-gray-50 h-28 dark:bg-gray-800">
-                          <p className="text-2xl text-gray-400 dark:text-gray-500">
-                              <svg className="w-3.5 h-3.5" aria-hidden="true" xmlns="http://www.w3.org/2000/svg"
-                                   fill="none" viewBox="0 0 18 18">
-                                  <path stroke="currentColor" strokeLinecap="round" strokeLinejoin="round"
-                                        strokeWidth="2" d="M9 1v16M1 9h16"/>
-                              </svg>
-                          </p>
-                      </div>
-                  </div>
+            <div className="flex items-center justify-center h-48 mb-4 rounded bg-gray-50 dark:bg-gray-800">
+                <p className="text-2xl text-gray-400 dark:text-gray-500">
+                    My Profile
+
+                </p>
+            </div>
+
+            <div className="flex items-center justify-center h-48 mb-4 rounded bg-gray-50 dark:bg-gray-800">
+                <p className="text-2xl text-gray-400 dark:text-gray-500">
+                    {session.user?.name }
+                    <br/>
+
+                    {session.user?.email}
+                    <br/>
+
+                    {session.user?.id}
+                    <br/>
```

Comment:
Just noting that these `<br/>` will render whether or not each field is present -- not sure if you want to tie them to rendering only if a given field is present or not.

Reply:
@aaronkanzer This will be updated in next iteration. I just placed this to test.